src/sublime_aio.py

```diff
@@ -106,13 +132,13 @@ def _debounced_callback(view: sublime.View, coro_args: tuple) -> None:
             if call_at[view.view_id] <= __loop.time():
                 del call_at[view.view_id]
                 if view.is_valid():
-                    coro_func, self, args, kwargs = coro_args
-                    __loop.create_task(coro_func(self, *args, **kwargs))
+                    # Tasks are collected later using `asyncio.all_tasks`.
+                    _ = __loop.create_task(coro_func())
                 return
 
-            __loop.call_at(call_at[view.view_id], _debounced_callback, view, coro_args)
+            __loop.call_at(call_at[view.view_id], _debounced_callback, view, coro_func)
 
-        def wrapper(self, *args, **kwargs) -> None:
+        def wrapper(self: sublime_plugin.EventListener | sublime_plugin.ViewEventListener, *args: sublime.View) -> None:
```

Comment:
I'd proably also point to `sublime_aio` Listeners here.

```suggestion
        def wrapper(self: EventListener | ViewEventListener, *args: sublime.View) -> None:
```

Also assigning `*args` the `sublime.View` type feels odd as it is possibly at most only the first argument, which is of that type.

Reply:
`sublime_aio` listeners have been adopted.

The `*args: sublime.View` was just the only idea I had to get the signature to capture both the EventListener and the ViewEventListener case with that argument that is only present in one case. I'm open for different ideas, but for now I find this acceptable.

Reply:
Just wondered it's working.